SU-HSDO/suhumsci
# SHS-5792: Person Full Name/Auto Entity Label bug

# READY FOR REVIEW

## Summary
* Fixes auto Full Name for the Person content type - when entering only a First and Last Name and saving, now the Full Name is created correctly and not a random string of characters.

## Need Review By (Date)
2024-08-28

## Urgency
medium

## Steps to Test
1. Login to the site
2. Create a Person node
3. Enter a First Name and Last Name, but leave the Full Name field empty
4. Save the node
5. On the front-end, verify that the title is the first name + last name and no random characters appear

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208165007263139
